package.json

```diff
@@ -43,10 +54,21 @@
   },
   "devDependencies": {
     "aegir": "^12.3.0",
+    "babel-cli": "6.26.0",
+    "babel-core": "6.26.0",
+    "babel-loader": "7.1.2",
+    "babel-preset-flow-node": "2.0.1",
     "buffer-equal": "1.0.0",
     "chai": "^4.1.2",
     "dirty-chai": "^2.0.1",
-    "pre-commit": "^1.2.2"
+    "flow-bin": "^0.66.0",
+    "flow-copy-source": "^1.2.0",
+    "lint-staged": "^6.0.0",
+    "pre-commit": "^1.2.2",
+    "prettier": "1.10.2",
+    "rollup": "0.56.1",
```

Comment:
Why was `rollup` introduced / where does it get used?

Reply:
I wondered the same. I'm in the process of pushing an updated version of this PR. There I don't use the rollup module and it still seems to work.